### PR TITLE
New package: dracut-crypt-ssh-1.0.3

### PR DESCRIPTION
**Pull request by tastytea**
dracut initramfs module to start dropbear sshd during boot to enter LUKS passphrase remotely.

### CODE REVIEW

File: srcpkgs/dracut-crypt-ssh/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+depends="dracut dracut-network dropbear openssh"
+short_desc="A new initramfs infrastructure - enter LUKS passphrase remotely"
+maintainer="tastytea <tastytea@tastytea.de>"
+license="GPL-2"
+homepage="https://github.com/dracut-crypt-ssh/dracut-crypt-ssh"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=158d293a1f4dc9e47bd4b390879dca084b5705cc21daf2743f3020dffefd966b
+
+pre_build() {
+	sed -i 's/CFLAGS=/CFLAGS=-pie -fPIC /' modules/60crypt-ssh/helper/Makefile
```
Comment (Vaelatern) [inline srcpkgs/dracut-crypt-ssh/template]: Please don't hardcode custom values, instead set it equal to whatever the shell variable `${CFLAGS}` is

Comment (tastytea) [inline srcpkgs/dracut-crypt-ssh/template]: I ran into the Error: "non-PIE executable found in PIE build", so I did what was suggested in the [forum](https://forum.voidlinux.eu/t/creating-a-template-non-pie-executable-found-in-pie-build/1043/5).
How should I proceed instead?

Build log without the CFLAGS changes: [xbarf_dracut-crypt-ssh_non-PIE-error.txt](https://github.com/voidlinux/void-packages/files/478924/xbarf_dracut-crypt-ssh_non-PIE-error.txt)

Comment (Vaelatern) [inline srcpkgs/dracut-crypt-ssh/template]:
```
sed -i "s/CFLAGS=/CFLAGS=${CFLAGS}/" modules/60crypt-ssh/helper/Makefile
```

Comment (tastytea) [inline srcpkgs/dracut-crypt-ssh/template]: That does not work, still the no-PIE error.

[xbarf_dracut-crypt-ssh_non-PIE-error_2.txt](https://github.com/voidlinux/void-packages/files/478976/xbarf_dracut-crypt-ssh_non-PIE-error_2.txt)

Comment (Vaelatern) [inline srcpkgs/dracut-crypt-ssh/template]: Then add ${LDFLAGS} to the mix please.

Comment (tastytea) [inline srcpkgs/dracut-crypt-ssh/template]: That works. Many thanks!

**Pull request closed without merging**